src/components/tracked-controls-webxr.js

```diff
@@ -62,17 +62,24 @@ module.exports.Component = registerComponent('tracked-controls-webxr', {
   },
 
   emitButtonDownEvent: function (evt) {
-    if (!this.controller || evt.inputSource.handedness !== this.data.hand) { return; }
-    if (this.controller.gamepad) { return; }
+    if (!this.controller || this.controller.gamepad) { return; }
```

Comment:
Maybe factor out this code and the one below in to a method `isControllerPresent`?

Reply:
I think if we factor this code out the PR is ready to go. Thanks for the patience.

Reply:
okay, yes I think I can do so!